app/screens/MachineDetails.js

```diff
@@ -221,6 +221,16 @@ class MachineDetails extends Component {
                                             () => this.setState({ showAddConditionModal: true }) :
                                             () => this.props.navigation.navigate('Login')}
                                     />
+                                    {location.operator_id ?
```

Comment:
We can avoid the ternaries returning `null` if we are certain we are using booleans for the check.
```suggestion
                                    {!!location.operator_id &&
                                        operatorHasEmail ?
                                            <View style={[s.operatorEmail,s.operatorHasEmail]}>
                                                <Text style={{textAlign:'center',color:theme.text2}}>This operator receives machine comments!</Text>
                                            </View> :
                                            <View style={[s.operatorEmail,s.operatorNotEmail]}>
                                                <Text style={{textAlign:'center',color:theme.white}}>This operator does not receive machine comments</Text>
                                            </View>
                                    }
```